app/src/main/res/values-it/strings.xml

```diff
@@ -308,33 +308,34 @@ ed anonima"</string>
     <string name="meldungen_meldung_more_button">"Scopri di più"</string>
     <string name="tracing_active_title">"Tracciatura attiva"</string>
-    <string name="tracing_active_text">"Gli incontri vengono salvati in modo anonimo sul tuo dispositivo."</string>
+    <string name="tracing_active_text">"Le co-presenze vengono conteggiate in modo anonimo sul tuo dispositivo."</string>
     <string name="tracing_error_title">"Tracciatura non attiva"</string>
-    <string name="tracing_error_text">"Attiva la tracciatura e il Bluetooth in modo che gli incontri possano essere registrati in modo anonimo."</string>
+    <string name="tracing_error_text">"Attiva la tracciatura e il Bluetooth in modo che le co-presenze possano essere registrate in modo anonimo."</string>
 
     <!-- Meldungen: zusätzliche Info Titel -->
     <string name="meldungen_additional_info_title">"Cosa significa?"</string>
 
     <!-- Meldungen: Info Text 1 -->
-    <string name="meldungen_additional_info_text1">"Per interrompere la diffusione di un virus, è necessario interrompere la catena di infezione. Puoi farlo isolando le persone contagiose. C'è però un problema con il virus COVID-19: potresti essere contagioso prima di sentirti male! Ciò consente al virus di diffondersi, anche se le persone con sintomi si isolano questo perché siamo un passo indietro."</string>
+    <string name="meldungen_additional_info_text1">"Per interrompere la diffusione di un virus, è necessario interrompere la catena d'infezione. Puoi farlo isolando le persone contagiose. C'è un problema con il virus della COVID-19:
+potresti essere contagioso prima di avvertirne i sintomi o in caso fossi asintomatico! Ciò consente al virus di diffondersi se non ci confiniamo, in tempo più breve possibile. Cosi facendo, il nostro sistema sanitario risulterebbe essere sempre un passo indietro per la lotta contro la COVID-19."</string>
 
     <!-- Meldungen: zusätzlicher Info Text -->
-    <string name="meldungen_additional_info_text2">"Pertanto le persone che sono entrate in contatto con chi è risultato positivo alla COVID-19 devono essere informate. Ciò consente loro di isolarsi presto prima di sperimentare sintomi evitando così di contagiare altre persone. Ovviamente, non tutti i contatti hanno il virus, ma in questo modo quelli che ce l'hanno non lo trasmettono. Così saremo un passo avanti rispetto al virus."</string>
+    <string name="meldungen_additional_info_text2">"Pertanto, le persone che sono state in co-presenza con chi è risultato positivo alla COVID-19 devono essere informate. Ciò consente loro di confinarsi presto prima di avvertire sintomi, evitando cosi di contagiare altre persone. Ovviamente, NON tutte le persone con chi sono stato in co-presenza saranno necessariamente contagiate, ma in questo modo quelli che sono effettivamente portatori non lo possono trasmettere se sono confinati. Cosi facendo, il nostro sistema sanitario sarà sempre un passo avanti rispetto alla COVID-19."</string>
```

Comment:
alla COVID-19  -->  al COVID-19

evitando cosi --> evitando così (i accentata)

ma in questo modo quelli che sono effettivamente portatori non lo possono trasmettere se sono confinati --> ma gli effettivi portatori confinandosi non lo possono trasmettere.

Reply:
> alla COVID-19 --> al COVID-19
> come sopra
> evitando cosi --> evitando così (i accentata)
> corretto
> ma in questo modo quelli che sono effettivamente portatori non lo possono trasmettere se sono confinati --> ma gli effettivi portatori confinandosi non lo possono trasmettere.
corretto.